tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1406,3 +1406,43 @@ def test_verify_attribute_of_users_are_updated(session, ldap_data, ldap_tear_dow
         assert ldap_data['ldap_user_name'] in user_values['user']['firstname']
         assert ldap_data['ldap_user_name'] in user_values['user']['lastname']
         assert ldap_data['ldap_user_name'] in user_values['user']['mail']
+
+
+@pytest.mark.parametrize(
+    "auth, username, password",
+    [
+        (pytest.lazy_fixture('auth_source_ipa'), settings.ipa.user_ipa, settings.ipa.password_ipa),
+        (pytest.lazy_fixture('auth_source'), settings.ldap.username, settings.ldap.password),
+    ],
+)
+@tier2
+def test_login_failure_if_internal_user_exist_IPA_AD(
```

Comment:
Seem this is mirror of `test_login_failure_rhsso_user_if_internal_user_exist`, except this is for IPA and other one is for RHSSO. Is that correct?

The other test is still hidden behind `skip_if_open`. Could you add this decorator here, too? Or remove it from other test, if it's not needed?

Reply:
Not needed. RHSSO uses external authentication method to login. These tests are plain vanilla LDAP authentication which is working fine hence I did not included the skip_if_open